linux/Platform.c

```diff
@@ -21,12 +21,16 @@ in the source distribution for its full text.
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
+#include <sys/ptrace.h>
 #include <sys/sysmacros.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
+#include "BacktraceScreen.h"
 #include "BatteryMeter.h"
+#include "CPUMeter.h"
 #include "ClockMeter.h"
 #include "Compat.h"
-#include "CPUMeter.h"
```

Comment:
The original sort order (binary collation) was fine … No need to touch this.

Reply:
I'll revert this change, thanks.